application/src/test/java/org/opentripplanner/transit/model/_data/TimetableRepositoryForTest.java

```diff
@@ -234,6 +235,13 @@ public List<StopTime> stopTimesEvery5Minutes(int count, Trip trip, int startTime
       .toList();
   }
 
+  /**
+   * @see TimetableRepositoryForTest#stopTimesEvery5Minutes(int, Trip, int)
+   */
+  public List<StopTime> stopTimesEvery5Minutes(int count, Trip trip, String startTime) {
```

Comment:
I think you should replace the old method. Remember to always check if you can replace another method if you "enhance it".

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/6250/commits/6b2cc822b43e7f7e3ee994b74b6ba71265fe43bb